util/uvmdvgen.py

```diff
@@ -50,19 +53,29 @@ def main():
                               DV library instead. (ignored if -e switch is not passed)"""
     )
 
+    parser.add_argument(
+        "-hr",
+        "--has_ral",
+        default=True,
+        action='store_false',
+        help="""Specify whether CIP has a RAL model"""
```

Comment:
CIP -> the DUT has CSRs and hence needs a RAL model.

Reply:
Updated.